docs/concepts/Security-Best-Practices.md

```diff
@@ -163,6 +163,14 @@ To learn more about secret scanning, see [About secret scanning](https://docs.gi
 
 To author sign a package, see [Sign a package](../create-packages/sign-a-package.md).
 
+### Reproducible Builds
+
+**📦🖊 Package Author**
+
+Reproducible builds creates binaries that are binary identical each time you build it, and contain source code links and compiler metadata that enable a package consumer to recreate the binary directly and validate that the build environment has not been compromised.
```

Comment:
Can we use the term "package" instead of "binary"? This may be more approachable to newbies that are familiar with NuGet packages but not binaries.

```suggestion
Reproducible builds create packages that are identical each time you build it, and contain source code links and compiler metadata that enable a package consumer to recreate the package directly and validate that the build environment has not been compromised.
```

Reply:
The issue is that technically the package is not deterministic but the binary is. Deterministic packages had issues with time stamps and deployment that I don't know if they were addressed?

Reply:
The term "binary" is consistent with reproducible builds. We should keep that here in my opinion. The binaries in the package are reproducible today, not the package itself afaik.

Reply:
Good point @clairernovotny, those issues have not been addressed yet.